client/src/test/java/au/org/ala/names/ws/client/ALANameUsageMatchServiceClientTest.java

```diff
@@ -0,0 +1,73 @@
+package au.org.ala.names.ws.client;
+
+import au.org.ala.names.ws.api.NameSearch;
+import au.org.ala.names.ws.api.NameUsageMatch;
+import au.org.ala.util.TestUtils;
+import okhttp3.mockwebserver.MockResponse;
+import okhttp3.mockwebserver.MockWebServer;
+import okhttp3.mockwebserver.RecordedRequest;
+import org.gbif.rest.client.configuration.ClientConfiguration;
```

Comment:
Not sure on the rationale of using these artifacts from GBIF. It would be nice to isolate this library from these...

Reply:
It does mean that we'll have to have something that parallels org.gbif.rest.client which is where we've been pulling retrofit functionality from. The ALANameUsageMatchServiceClient uses this package to fit out the client and talk to OkHttp.

Reply:
I can live that. I'd prefer that to having this dependency.

Reply:
It shall be done.

Reply:
I'll try this first and see if it removes the okio dependency, as well.

Reply:
Dependencies removed. With new! bonus! test cases.